ci/create-sencha-environment.sh

```diff
@@ -31,23 +31,23 @@ mkdir -p $INSTALL_DIR
 cd $DOWN_DIR
 
 # DOWNLOAD (if needed)
-# 1) Sencha cmd (v6.0.0.202)
+# 1) Sencha cmd (v6.2.0.103)
 if [ ! -f "SenchaCmd-$SENCHA_CMD_VERSION-linux-amd64.sh.zip" ]; then
     wget "http://cdn.sencha.com/cmd/$SENCHA_CMD_VERSION/no-jre/SenchaCmd-$SENCHA_CMD_VERSION-linux-amd64.sh.zip"
 fi
 
-# 2) Ext JS (v6.0.0.640)
+# 2) Ext JS (v6.2.0.103)
 if [ ! -f "ext-$SENCHA_EXTJS_VERSION-gpl.zip" ]; then
     wget "http://cdn.sencha.com/ext/gpl/ext-$SENCHA_EXTJS_VERSION-gpl.zip"
 fi
 
 # EXTRACT (if needed)
-# 1) Sencha cmd (v6.0.0.202)
+# 1) Sencha cmd (v6.2.0.103)
 if [ ! -f "SenchaCmd-$SENCHA_CMD_VERSION-linux-amd64.sh" ]; then
```

Comment:
I think the `*.sh` file is name like this:

`SenchaCmd-6.2.0.103-linux-amd64.sh`

We only have the `6.2.0` part in the script.

I think it'd be good if we introduced

```bash
SENCHA_CMD_VERSION_FULL="6.2.0.103"
```

and changed that line to read

```bash
if [ ! -f "SenchaCmd-$SENCHA_CMD_VERSION_FULL-linux-amd64.sh" ]; then
```